test/HealthChecks.ArangoDb.Tests/HealthChecks.ArangoDb.Tests.csproj

```
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>$(NetFrameworkVersion)</TargetFramework>
+    <TargetFrameworks>$(NetFrameworkVersionLts);$(NetFrameworkVersionCurrent)</TargetFrameworks>

```

```suggestion
    <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
```

for all test projects